### PR TITLE
Added script to configure BGP for l3bgp-demo target.

### PR DESCRIPTION
**Pull request by gkvijay**
Makefile target for L3 mode demo setup without BGP.
make l3bgp-demo : sets up routing mode, configures BGP
make l3-demo : routing mode without BGP (no quagga vms)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -233,10 +233,15 @@ l3-test:
 	cd $(GOPATH)/src/github.com/contiv/netplugin/scripts/python && PYTHONIOENCODING=utf-8 ./createcfg.py -contiv_l3 2
 	CONTIV_L3=2 CONTIV_NODES=3 go test -v -timeout 900m ./test/systemtests -check.v -check.abort
 	CONTIV_L3=2 CONTIV_NODES=3 make stop
-l3-demo:
+
+l3-demo: demo
```
Comment (rchirakk) [inline Makefile]: Could you add a line that it is for  docker/swarm  ?

File: scripts/l3bgp_demo.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
+
```
Comment (rchirakk) [inline scripts/l3bgp_demo.sh]: it is better to keep this under docs dir.
And ```netctl``` can be moved to Makefile itself

File: Makefile (hunk 1 above)
Comment (gkvijay) [inline Makefile]: done

File: scripts/l3bgp_demo.sh (hunk 2 above)
Comment (gkvijay) [inline scripts/l3bgp_demo.sh]: This is just a description of the l3bgp-demo setup. BGP documentation is already present in contiv doc/tutorial.

**Comment by rchirakk** [comment]
LGTM

**Comment by g1rana** [comment]
LGTM

**Pull request merged**